spec/calendar.html

```diff
@@ -152,14 +156,16 @@ <h1>
       <dl class="header">
         <dt>description</dt>
         <dd>
-          It adds _duration_ to _date_ by calling the `dateAdd` method of the given _calendar_.
+          It adds _duration_ to _date_ by calling the `dateAdd` method of the given _calendar_, which may be a built-in calendar.
```

Comment:
Should this say "...built-in calendar identifier" instead of just "built-in calendar" here and below?  Might be clearer to globally clarify when we say "calendar" or "time zone" that we mean an object, a built-in string identifier, or either.